src/client/unittests/nosetest/collector.ts

```diff
@@ -1,12 +1,12 @@
 'use strict';
-import * as path from 'path';
-import { execPythonFile } from './../../common/utils';
-import { TestFile, TestSuite, TestFunction, Tests } from '../common/contracts';
 import * as os from 'os';
-import { extractBetweenDelimiters, convertFileToPackage, flattenTestFiles } from '../common/testUtils';
+import * as path from 'path';
```

Comment:
Do you have a rule-of-thumb that you follow for when you import a module instead of specific objects from a module?

Reply:
Yes, best practice is to import what you need. Import * is a carry on from old code (importing specifics was added later).

Reply:
My plan going forward is to be explicit (at least when importing internal modules)

Reply:
Yeah, I just wasn't sure what the common practice is since I personally import to the module in Python and then access from there.